terraform/workspace_variables/staging.tfvars

```diff
@@ -18,3 +18,9 @@ statuscake_alerts = {
     node_locations = ["UKINT", "UK1", "MAN1", "MAN5", "DUB2"]
   }
 }
+
+#vault
+key_vault_name           = "s121t01-shared-kv-01"
```

Comment:
The alignment doesn't look correct. Have you run terraform fmt?

Reply:
all ready done..

Reply:
Please run it again then